tests/API/OVAL/unittests/test_state_check_existence.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+result=`mktemp`
+
+set -e
+set -o pipefail
+
+cat > /tmp/foo.txt <<EOF
```

Comment:
What if somebody else decided to use same file? I would consider to use the ```mktemp``` here